src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/VSCodeAnalyzerLoader.cs

```diff
@@ -25,21 +26,21 @@ public void InitializeDiagnosticsServices()
     {
     }
 
-    public static IAnalyzerAssemblyLoader CreateAnalyzerAssemblyLoader(ExtensionAssemblyManager extensionAssemblyManager, ILogger logger)
+    public static IAnalyzerAssemblyLoader CreateAnalyzerAssemblyLoader(Workspace workspace, ExtensionAssemblyManager extensionAssemblyManager, ILogger logger)
     {
-        return new VSCodeExtensionAssemblyAnalyzerLoader(extensionAssemblyManager, logger);
+        var analyzerLoaderProvider = workspace.Services.GetRequiredService<IAnalyzerAssemblyLoaderProvider>();
```

Comment:
can you pass the services along instead, instead of the Workspace itself?

Reply:
alternatively, could VSCodeAnalyzerLoader become a workspace service itself (it woudl tehn be passed the services when constructed).